src/main/java/net/imagej/legacy/ImageJ2Options.java

```diff
@@ -84,6 +84,16 @@
 		callback = "run")
 	private boolean sciJavaIO = false;
 
+	/**
+	 * 
+	 */
```

Comment:
Please get rid of this empty javadoc block, and reformat the `@Parameter` according to the ImageJ code style.